src/entity.ts

```diff
@@ -21,6 +21,7 @@ import {PathType} from '.';
 import {protobuf as Protobuf} from 'google-gax';
 import * as path from 'path';
 import {google} from '../protos/protos';
+import {AND, PropertyFilter} from './filter';
```

Comment:
nit, upper case is usually reserved for variable constants `const FOO = 9;  BAR = 99`, to make it clear that this is an exported method, I might go with:

`and`/`or`,  or `applyAnd`, `applyOr`.

Reply:
Choosing the former